site/docs/components/index.mdx

```diff
@@ -20,7 +20,7 @@ with designs for light and dark modes in all four densities.
 
 Stable components do not feature an accompanying symbol in their navigation link titles. You can find them in the `@salt-ds/core` package.
 
-The {meta.labsIcon} symbol denotes components that are under active development. These may not function as expected and may change due to feedback. You should exercise caution. You can find them in the `@salt-ds/lab` package.
+The {meta.labsIcon} symbol denotes components that are under active development. These may not function as expected and may change due to feedback. You should exercise caution. You can find them in the `@salt-ds/lab` package. Please don't use lab components not listed on the website, which aren't receiving active development.
 
```

Comment:
maybe this? we use the wording in a few places, whatever sound less encouraging i guess 
```suggestion
The {meta.labsIcon} symbol denotes components that are under active development. These may not function as expected and may change due to feedback. You should exercise caution. You can find them in the `@salt-ds/lab` package. The use of lab components not listed on the website is not encouraged since they aren't receiving active development.

```